card/card.gemspec

```diff
@@ -17,37 +13,49 @@ Gem::Specification.new do |s|
   s.license       = 'GPL'
 
   s.files         = `git ls-files`.split($/)
-  
+
+  # add submodule files (seed data)
+  `git submodule --quiet foreach pwd`.split($\).each do |submod_path|
+    gem_root = File.expand_path File.dirname(__FILE__)
+    relative_submod_path = submod_path.gsub "#{gem_root}/", ''
+    Dir.chdir(submod_path) do
+      s.files += `git ls-files`.split($\).map do |filename|
+        "#{relative_submod_path}/#{filename}"
+      end
+    end
+  end
```

Comment:
Sure would be nice if there were some git helpers for gemspecs that would do this with a simpler interface.  Ugly but effective.